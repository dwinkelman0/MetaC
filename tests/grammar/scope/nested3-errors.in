int func() {
    syntax error(3);
    for (int i = 0; i < 3; i += 1) {
        Object_t obj = get(i);
        obj.things = ;
    }
    return 0;
}