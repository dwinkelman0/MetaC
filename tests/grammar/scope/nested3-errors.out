int func() {
    for (int i = 0; i < 3; i += 1) {
        Object_t obj = get(i);
    }
    return 0;
}